packages/algob/src/types.ts

```diff
@@ -363,7 +377,7 @@ export interface AlgobDeployer {
   putMetadata: (key: string, value: string) => void
   getMetadata: (key: string) => string | undefined
   deployASA: (name: string, flags: ASADeploymentFlags) => Promise<ASAInfo>
-  deployASC: (name: string, source: string, account: Account) => Promise<ASCInfo>
+  deployASC: (name: string, sc_params: Object, flags: ASCDeploymentFlags) => Promise<ASCInfo>
```

Comment:
referring to the comment above, we are missing one Object parameter here.

Reply:
ok, i will make two parameters